addons/account_tax_python/models/account_tax.py

```diff
@@ -8,7 +8,9 @@
 class AccountTaxPython(models.Model):
     _inherit = "account.tax"
 
-    amount_type = fields.Selection(selection_add=[('code', 'Python Code')])
+    amount_type = fields.Selection(selection_add=[
+        ('code', 'Python Code')
+    ], ondelete={'code': 'cascade'})
```

Comment:
This does not match the uninstall hook:
```suggestion
    ], ondelete={
        'code': lambda recs: recs.write({'amount_type': 'percent', 'active': False})
    })
```

Reply:
done